core/consensus/src/tests/engine.rs

```diff
@@ -23,11 +23,12 @@ use crate::engine::ConsensusEngine;
 use crate::fixed_types::FixedPill;
 use crate::status::StatusAgent;
 use crate::util::OverlordCrypto;
-use crate::wal::SignedTxsWAL;
+use crate::wal::{ConsensusWal, SignedTxsWAL};
 
 use super::*;
 
-static FULL_TXS_PATH: &str = "./free-space/engine";
+static FULL_TXS_PATH: &str = "./free-space/engine/txs";
```

Comment:
Save them all together in the `data_path` directory? This allows us to view all our data in one unified directory.

Reply:
no, that will confuse